src/components/FormInput/index.js

```diff
@@ -15,6 +16,17 @@ const Input = ({ input, meta: { error }, label, type, className, placeholder })
     />
     {error && <span>{error}</span>}
   </div>
-  )
+)
+
+Input.propTypes = {
+  input: PropTypes.object,
+  label: PropTypes.string,
+  type: PropTypes.string,
+  className: PropTypes.string,
+  placeholder: PropTypes.string,
+  meta: PropTypes.shape({
+    error: PropTypes.string,
+  }),
```

Comment:
And here: https://github.com/erikras/redux-form/blob/master/src/propTypes.js#L84